### PR TITLE
NavBar & About Me sections

### PR DESCRIPTION
**Pull request by MattW-R**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+	font-size: 22px;
+	font-family: 'Orbitron', sans-serif;
+	font-weight: 600;
+}
+
+body {
+	background-color: #120458;
+	color: #FE75FE;
+	margin-top: 60px;
+	background-image: url(BG.png);
```
Comment (khorh) [inline styles.css]: Create an images folder for longevity

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+	background-position: center;
+	background-attachment: fixed;
+}
+
+nav {
+	position: fixed;
+	background-color: #120458;
+	top: 0;
+	width: 100%;
+	color: #FF00A0;
```
Comment (khorh) [inline styles.css:27]: Just a recommendation... depending on what accessibility level you are after, this colour does not pass AAA standards in normal text.  Your nav text is larger so this is actually fine.

https://webaim.org/resources/contrastchecker/

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		<title>Matt W-R Portfolio</title>
+		<link rel="preconnect" href="https://fonts.gstatic.com" />
+		<link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Orbitron:wght@600&display=swap" rel="stylesheet" />
+		<link rel="stylesheet" type="text/css" href="normalize.css" />
+		<link rel="stylesheet" type="text/css" href="styles.css" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	</head>
+	<body>
+		<nav>
+			<div>
```
Comment (khorh) [inline index.html]: There seems to be an extra \<div>.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+p {
+	font-size: 0.8rem;
+	font-weight: 400;
+	font-family: 'Montserrat', sans-serif;
+}
+
+nav a {
+	display: inline-block;
+	text-decoration: none;
+	color: #FF00A0;
```
Comment (khorh) [inline styles.css]: Color has been specified earlier in the nav tag (line 28)

**Review by khorh — COMMENTED** [review]
Completed review

File: styles.css (hunk 2 above)
Comment (MattW-R) [reply styles.css:27]: Thank you for recommendation. I will not be using this colour in normal text. However, I have now changed the hover colour in my About Me section (to white) to meet this recommendation.

Comment (khorh) [reply styles.css:27]: > Thank you for recommendation. I will not be using this colour in normal text. However, I have now changed the hover colour in my About Me section (to white) to meet this recommendation.

Great!

**Pull request merged**